Update: hyperdrive-daemon integration (#153)
p2pcommons/sdk-js: README.md

Returns a new instance of the sdk.

- `opts` can include the following:

```javascript
{
  baseDir: // defaults to '~/.p2pcommons'

>   rootKey: // archive key. Used when more than one sdk instance wants to share an hyperdrive custom folder

Reviewer: is it a root key or an archive key? :P

Reviewer: what is a "hyperdrive custom folder"?

>   memory: // A boolean that indicates if modules should be stored in-memory. Defaults to `false`

Reviewer: ```suggestion
  memory: // A boolean that indicates if modules should be stored in-memory only. Defaults to `false`
```

>   storage: // Indicates a path to the hyperdrives folder. Defaults to the `hyperdrive-daemon` default storage.

Reviewer: ```suggestion
  storage: // Path to the hyperdrives folder. Defaults to the `hyperdrive-daemon` default storage location.
```

>   disableSwarm: // Used to avoid announcing our modules to the DHT, preventing them to be `seeded`. Defaults to `false`

Reviewer: ```suggestion
  disableSwarm: // Used to disable announcing our modules to the DHT, preventing them to be "seeded". Defaults to `false`
```

```javascript
{
  baseDir: // defaults to '~/.p2pcommons'
  rootKey: // archive key. Used when more than one sdk instance wants to share an hyperdrive custom folder

>   persist: // Indicates if the drives should be persisted or stored in-memory. Defaults to `true`

Reviewer: This is misleading, it could also be interpreted that `true` will make it so the drive will be persisted or stored in memory.

Reviewer: I think this was better before

Author: I can rephrase it to something like: Boolean that indicates wether to save your modules into disk (true) or in-memory (false). Defaults to `true`

Or just change it to a `memory: true|false` option. To switch into in-memory storage.

Reviewer: if persistence is disabled, it's obvious that state will be stored in memory, as that's what all programs do. So I don't think it needs to be mentioned.

  storage: // Indicates a path to the hyperdrives folder. Defaults to the `hyperdrive-daemon` default storage.

>   disableSwarm: // defaults to undefined. If `false` will set the `hyperdrive-daemon` `noAnnounce` flag.

Reviewer: Whether `noAnnounce` will be set is an implementation detail, can we describe what this _means_ to the user?

- hash: represents the key (`url`) to be looked for. It is the buffer archive key `.toString('hex')`

Returns an object with:

> 

Reviewer: I love that except for the constructor the API staid the same <3
